### ISSUE TITLE
Bug when upgrading to 3.6.16 from 3.6.12

### ISSUE THREAD

**Issue by antoniopaisfernandes** [HuasoFoundries/jpgraph#56]
When my autodeploy script updated jpgrah to the latest, it started displaying errors:

```
ErrorException
Use of undefined constant ROOT_PATH - assumed 'ROOT_PATH'
```

line 23 of src/config.inc.php.

Best regards

**Comment by amenadiel** [comment]
@antoniopaisfernandes You're right! I fell short of one folder level when inferring the project root folder.

I fixed this and tagged it as 3.6.17

**Comment by antoniopaisfernandes** [comment]
Thanks
